### PR TITLE
[14.0][FIX] adapt syntax for act_window lookup

### PR DESCRIPTION
**Pull request by florentx**
Follow-up of  #1325 

It fixes the potential AccessError for ~six~ five logistics add-ons where the same syntax was found.

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Comment by florentx** [comment]
I've reverted the change on `"stock_available_unreserved"` --> it was not an `act_window` xmlid

**Comment by sebalix** [comment]
@rousseldenis are you OK to merge this? (as it impacts some addons we aren't using ourself, I prefer to have a second review :) )

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**
